Docs/AcpiSamples/SSDT-PLUG.dsl

```diff
@@ -61,6 +63,36 @@ DefinitionBlock ("", "SSDT", 2, "ACDT", "CpuPlug", 0x00003000)
             }
         }
     }
+    
+    If (CondRefOf (\_PR.C000)) {
+        Scope (\_PR.P000) {
```

Comment:
Incorrect CPU name

Reply:
Ah ok I see what you mean, my bad will fix